api/routers/edr.py

```diff
@@ -27,40 +34,74 @@
 @router.get(
     "/locations",
     tags=["Collection data queries"],
-    response_model=FeatureCollection,
+    response_model=EDRFeatureCollection,
     response_model_exclude_none=True,
 )
 # We can currently only query data, even if we only need metadata like for this endpoint
 # Maybe it would be better to only query a limited set of data instead of everything (meaning 24 hours)
 async def get_locations(
     bbox: Annotated[str, Query(example="5.0,52.0,6.0,52.1")]
-) -> FeatureCollection:  # Hack to use string
+) -> EDRFeatureCollection:  # Hack to use string
```

Comment:
Why do we need a custom FeatureCollection? Is regular geojson not sufficient?

Reply:
The focus of regular GeoJSON is representing spatial data. EDR GeoJSON adds custom properties that help users discover what is available in the request. See here for the [optional requirements](https://docs.ogc.org/is/19-086r4/19-086r4.html#toc9).

The GeoJSON package implements the GeoJSON standards. The models inside the package ignore extra properties. Therefore it cannot be used to add the optional properties of [EDR Feature Collection GeoJSON](https://schemas.opengis.net/ogcapi/edr/1.1/openapi/schemas/edr-geojson/edrFeatureCollectionGeoJSON.yaml). In addition, I think it is nice to have validation on the parameters which a custom model makes possible.

I do believe this is not the right place for this model, I think a better location would be https://github.com/KNMI/edr-pydantic. I will make a PR on that repository to add the model.

Reply:
Yes, I think this can be used for other endpoints too.

Reply:
I am not sure the `/items` parameter needs the `parameters` field in the FeatureCollection, as the Features returned are timeseries, which basically are parameters. So it would be duplicate in some sense.

Reply:
Aha, so this is a parameter field for a FeatureCollection, which is unnecesarry for `/items` since that `FeatureCollection` only contains `Features`. Sorry I am a bit slow here.

Reply:
Let's see what makes the most sense for the different use of GeoJSON in the E-SOH API. :-)
@Jeffrey-Vervoort-KNMI, I agree that it makes sense to have this in the edr-pydantic package, but maybe best if we move that once we have the E-SOH side working, so we don't have to make multiple iterations?

Reply:
That is fine for me, then we merge this one with the custom class, and update the edr-pydantic package later on.